source/core/Events.cpp

```diff
@@ -1048,6 +1048,9 @@ VIREO_FUNCTION_SIGNATUREV(WaitForEventsAndDispatch, WaitForEventsParamBlock)
         Int32 &staticCount = eventInfo->eventStructInfo[eventStructIndex].setCount;
 
         UInt32 msTimeout = timeOutPtr ? *timeOutPtr : -1;
```

Comment:
I'll delegate to @cglzaguilar @spathiwa @sanmut for these changes.

@amehra-ni make sure to turn on GitHub Actions in your Fork so you can see the CI builds in your branch. It currently does not look like your fork is testing your branches. You should see the GitHub Action configuration in: https://github.com/amehra-ni/VireoSDK/settings/actions

@amehra-ni before the PR is merged make sure to [run the squad tests](https://ni.visualstudio.com/DevCentral/_wiki/wikis/AppCentral.wiki/2445/How-to-update-Vireo-in-NXG?anchor=trigger-the-web-module-squad-tests) and link to a passing build in the PR description.

Marking approve pending the other CODEOWNERS feedback